lib/matplotlib/backends/backend_wx.py

```diff
@@ -618,7 +618,8 @@ def gui_repaint(self, drawDC=None, origin='WX'):
                else self.bitmap)
         drawDC.DrawBitmap(bmp, 0, 0)
         if self._rubberband_rect is not None:
-            x0, y0, x1, y1 = self._rubberband_rect
+            # Some versions of wx+python don't support numpy.float64 here.
+            x0, y0, x1, y1 = map(round, self._rubberband_rect)
```

Comment:
In Qt, it appears we use `int` instead of `round`.

Reply:
sure.